e2e/rbd.go

```diff
@@ -196,9 +196,13 @@ var _ = Describe("RBD", func() {
 		if err != nil {
 			e2elog.Failf("failed to create configmap with error %v", err)
 		}
-		err = createRBDStorageClass(f.ClientSet, f, defaultSCName, nil, nil, deletePolicy)
-		if err != nil {
-			e2elog.Failf("failed to create storageclass with error %v", err)
+		// Since helm deploys storageclass, skip storageclass creation if
+		// ceph-csi is deployed via helm.
+		if !helmTest {
+			err = createRBDStorageClass(f.ClientSet, f, defaultSCName, nil, nil, deletePolicy)
```

Comment:
there are many calls to `createRBDStorageClass()` (after it has been deleted), it is surprising that this isn't a problem. I guess the default values match up everywhere.